### ISSUE TITLE
ID 257492: [Programmatic Access - Simon - Variables>New]: Visual label and aria label is not same for 'Update' button under "Variables>Edit" screen.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#66334]
### **User Experience:** 
Visually impaired users who depend on screen readers will get disoriented and will face difficulty in understanding the control if correct `aria-label` and visual label do not match each other. 
_Note: User credentials should NOT be included in the bug._

### **Repro Steps:**

1. Open #URL: [play.grafana.org](https://play.grafana.org/).  Sign in with valid credentials.  "Home" page appears.
2. Navigate the Feature Showcase pane heading and select it.  
3. Navigate the expanded Dropdown and select the "Edit" menu item.  
4. Navigate the page "Edit Panel" page which appears.  
5. Navigate to "Settings" button and select it.  
6. Navigate the "Variables" tab item and select it.  
7. Navigate the "Variables" tab and "New" button. 
8. Navigate the "Variables edit" page.  
9. Verify the issue with screen reader.

**Actual Result:** 
Issue: Visual `label` and `aria label` is not same for 'Update' button under "Variables>Edit" screen.
Observation: Screen reader is reading as "Variable editor Submit button" when focus moves on 'Update' button.

**Expected Result:**
Visual `label` and `aria label` should be same as "Update button" for 'Update' button under "Variables>Edit" screen.
_Note:  The same issue exists with NVDA and JAWS._

### **Severity:**
`SEV2`

### **Area:**
`Dashboards Settings`

**Comment by evictorero** [comment]
Fixed on https://github.com/grafana/grafana/commit/56aaea4a8a2b96434eb85462f234998dfe307c0a
